src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  * <p>
  *   If even this mechanism does not provide a target file name, and the target file is deemed non-optional,
- *   the a {@link UserException.BadArgumentValue} is thrown.
+ *   the a {@link CommandLineException.BadArgumentValue} is thrown.
```

Comment:
Is this because the exception lives in a different project? We have many javadoc links to classes in different projects across the codebase, I believe.